BurnerB/FreeMentors-V2
# [#167962070] Failing signin tests

#### What does this PR do?
Failing tests for user signin
#### Description of Task to be completed?
* Write failing tests for user signup in accordance with TDD
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ch-failing-signin-tests-167962070 
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should fail

#### What are the relevant pivotal tracker stories?
[#167962070](https://www.pivotaltracker.com/story/show/167962070)

#### Screenshots (if appropriate)
![failing signin](https://user-images.githubusercontent.com/45785786/63342644-abe39a00-c354-11e9-9930-058358266395.png)
